### PR TITLE
LLVM 8 release

### PR DESCRIPTION
**Pull request by rohitaggarwal007**

### CODE REVIEW

File: src/test-suite/sanity_check.sh.cmake [hunk 1]
```diff
@@ -26,14 +26,11 @@ fi
 
 SEED_VERSION=$2
```
Comment (svkeerthy) [inline src/test-suite/sanity_check.sh.cmake:27]: Shall we remove $2 altogether?

**Review by svkeerthy — CHANGES_REQUESTED** [review]
sanity_check.sh.cmake needs fixing as discussed

File: src/test-suite/sanity_check.sh.cmake (hunk 1 above)
Comment (rohitaggarwal007) [reply src/test-suite/sanity_check.sh.cmake:27]: Remove the if clause for SEED_VERSION

**Comment by rohitaggarwal007** [comment]
> sanity_check.sh.cmake needs fixing as discussed

Done

**Review by svkeerthy — APPROVED** [review]

**Pull request merged**